src/relayserver/RelayServer.ts

```diff
@@ -37,10 +37,15 @@ import Timeout = NodeJS.Timeout
 const VERSION = '2.0.0-beta.3'
 const GAS_RESERVE = 100000
 
+// must receive some response for request from node
+const READY_TIMEOUT = 30000
```

Comment:
I think this should go to the configuration

Reply:
ok